.ci/data/cc3-ubuntu-xenial.yaml

```diff
@@ -0,0 +1,67 @@
+---
+base_image_url: https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
+base_image_name: Ubuntu 16.04
+...
+---
+...
+---
+{{- define "ENV" -}}
+{{proxyVars .}}
+{{- print " DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true " -}}
+{{end}}
+#cloud-config
+users:
+  - name: {{.User}}
+    gecos: CC Tester User
+    lock-passwd: true
+    shell: /bin/bash
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    ssh-authorized-keys:
+    - {{.PublicKey}}
+
+write_files:
+{{- if len $.HTTPProxy }}
+ - content: |
+     [Service]
+     Environment="HTTP_PROXY={{$.HTTPProxy}}"{{if len .HTTPSProxy}} "HTTPS_PROXY={{.HTTPSProxy}}{{end}}"{{if len .NoProxy}} "NO_PROXY={{.NoProxy}},{{.Hostname}}{{end}}"
+   path: /etc/systemd/system/docker.service.d/http-proxy.conf
+{{- end}}
+{{with proxyEnv . 5}}
+ - content: |
+{{.}}
+   path: /etc/environment
+{{end}}
+
+apt:
+{{- if len $.HTTPProxy }}
+  proxy: "{{$.HTTPProxy}}"
+{{- end}}
+{{- if len $.HTTPSProxy }}
+  https_proxy: "{{$.HTTPSProxy}}"
+{{- end}}
+package_upgrade: {{with .PackageUpgrade}}{{.}}{{else}}false{{end}}
+
+runcmd:
+ - {{beginTask . "Booting VM"}}
+ - {{endTaskOk . }}
+
+ - {{beginTask . (printf "Adding %s to /etc/hosts" .Hostname) }}
+ - echo "127.0.0.1 {{.Hostname}}" >> /etc/hosts
+ - {{endTaskCheck .}}
+
+ - echo "GOPATH=\"{{$.GoPath}}\"" >> /etc/environment
+ - echo "PATH=\"$PATH:/usr/local/go/bin:{{$.GoPath}}/bin\""  >> /etc/environment
+
+ - {{beginTask . "Downloading Go" }}
+ - {{download . "https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz" "/tmp/go1.8.3.linux-amd64.tar.gz"}}
```

Comment:
Is that possible to determine `https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz` or `go1.8.3.linux-amd64.tar.gz` or even the go version on the fly ? I mean it sounds better not to hardcode the go version because we want to rely on the latest stable go version.